src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userValidator = new UserService();
```

Comment:
wrong name

Reply:
Should be like this?
UserService userService = new UserService();